File: src/main/java/io/github/probeiuscorp/uf2m/data/UF2MBiomes.java
package io.github.probeiuscorp.uf2m.data;

import io.github.probeiuscorp.uf2m.data.dimension.meridian.BiomeMeridianMidnightZone;
import io.github.probeiuscorp.uf2m.data.dimension.meridian.BiomeMeridianWastes;
import net.minecraft.data.worldgen.BootstapContext;
import net.minecraft.resources.ResourceKey;
import net.minecraft.world.level.biome.Biome;

public class UF2MBiomes {
    public static final ResourceKey<Biome> MERIDIAN_WASTES = DBP.createBiomeKey("meridian_wastes");
    public static final ResourceKey<Biome> MERIDIAN_MIDNIGHT_ZONE = DBP.createBiomeKey("meridian_midnight_zone");

    public static void bootstrap(BootstapContext<Biome> context) {
        context.register(MERIDIAN_WASTES, BiomeMeridianWastes.createMeridianWastesBiome(context));
        context.register(MERIDIAN_MIDNIGHT_ZONE, BiomeMeridianMidnightZone.createMeridianMidnightZoneBiome(context));
    }
}

File: src/main/kotlin/io/github/probeiuscorp/uf2m/data/UF2MBiomes.kt
package io.github.probeiuscorp.uf2m.data

import io.github.probeiuscorp.uf2m.data.DBP.createBiomeKey
import io.github.probeiuscorp.uf2m.data.dimension.meridian.BiomeMeridianMidnightZone
import io.github.probeiuscorp.uf2m.data.dimension.meridian.BiomeMeridianWastes
import net.minecraft.data.worldgen.BootstapContext
import net.minecraft.resources.ResourceKey
import net.minecraft.world.level.biome.Biome

object UF2MBiomes {
  @JvmField
  val MERIDIAN_WASTES = createBiomeKey("meridian_wastes")
  @JvmField
  val MERIDIAN_MIDNIGHT_ZONE = createBiomeKey("meridian_midnight_zone")

  fun bootstrap(context: BootstapContext<Biome>) {
    context.register(MERIDIAN_WASTES, BiomeMeridianWastes.createMeridianWastesBiome(context))
    context.register(MERIDIAN_MIDNIGHT_ZONE, BiomeMeridianMidnightZone.createMeridianMidnightZoneBiome(context))
  }
}
